cudet/updates.py

```diff
@@ -0,0 +1,94 @@
+#    Copyright 2016 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+
+import subprocess
+import yaml
+import glob
+from copy import deepcopy
+from fuelclient.commands import base, environment
+from fuelclient.cli.actions.fact import DeploymentAction
+from fuelclient.cli.actions.settings import SettingsAction
+from fuelclient.cli.error import ServerDataException
+
+MU_UPGRADE_DICT_RESTRICTED = {
+    'mu_upgrade': {
+        'enabled': True,
+        'restart_rabbit': True,
+        'restart_mysql': True,
+    }
+}
+MU_UPGRADE_DICT_FULL = deepcopy(MU_UPGRADE_DICT_RESTRICTED)
+MU_UPGRADE_DICT_FULL['mu_upgrade'].update({
+    'metadata': {
+        'group': 'general',
+        'label': 'MU label',
+        'restrictions': [
+            {'action': 'hide'}
+        ],
+        'weight': 10
+    },
+    'type': 'hidden',
+    'value': True
+})
+
+
+class Updates(base.BaseCommand):
+    """TODO: provide meaningful description"""
+
```

Comment:
`TypeError: Can't instantiate abstract class Updates with abstract methods entity_name`
I cannot get it working...